runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DistributionCounterUpdateAggregator.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Lets move all the aggregators to `org/apache/beam/runners/dataflow/worker/counters`

Reply:
done.